p2p/p2p.go

```diff
@@ -354,3 +354,18 @@ func handleStream(stream net.Stream) {
 
 	log.Debug("got message", "peer", stream.Conn().RemotePeer(), "type", msgType, "msg", msg.String())
 }
+
+// Peers returns connected peers
+func (s *Service) Peers() []string {
+	return convIdToStringArray(s.host.Network().Peers())
+}
+
+// Network state
+func (s *Service) ID() string {
+	return s.host.ID().String()
+}
+
+// Should have peers
+func (s *Service) ShouldHavePeers() bool {
```

Comment:
This needs to be renamed. This should return the inverse of the `NoDiscovery` config iption